app/assets/javascripts/admin/views/user/experience_modal_view.js

```diff
@@ -1,21 +1,26 @@
 // @flow
 
 import _ from "lodash";
-import React from "react";
+import * as React from "react";
```

Comment:
This was autogenerated by the flowupdate tool. The organization of types and classes changed a bit, which is why this is recommended. Most of the time `import React` would work, but stuff like 
`React.ComponentType` is only accessible with the `import *` style. To solve some potential import headaches, it's probably a good idea to have the imports this way.